discuss/lib/discuss_web/templates/topic/new.html.heex

```diff
@@ -0,0 +1,7 @@
+<%= form_for @changeset, Routes.topic_path(@conn, :create),fn f ->  %>
+    <div class = "form-group">
+        <%= text_input f, :title, placeholder: "Title", class: "form-control"  %>
+        <% error_tag f, :title %>
+    </div>
+    <%= submit "Save Topic", class: "btn btn-primary" %>
+<%end%>
```

Comment:
Don't forget to add the empty line at the end of the file. Check the files with .heex ext.